focus_validator/rules/base_rule_definitions/FV-D010-0001.yaml

```
@@ -0,0 +1,4 @@
+check_id: FV-D010-0001
```

```suggestion
check_id: FV-M010-0001
```

Update to reflect metric or move everything to C010-000X.yaml

---

Changed this to FV-M001--0001